### PR TITLE
0.2.2.15 Tidy Code

### PR DESCRIPTION
**Pull request by sharrief**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
 This PR removes code from client and animal components/modules that were duplicated from Ticket related modules. 
It also removes unused Ticket related code in ClientService.

## Related Tickets & Documents

[0.2.2.15 Tidy Code](https://projects.digitalaidseattle.org/project-tasks-details?recordId=recWhsp2ZkCR7KlpF)

## QA Instructions, Screenshots, Recordings

Pages at /animals /client and /clients were updated with a "Not implemented yet" message which replaces the unrelated Ticket specific UI that was there.

### UI accessibility checklist

- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Color contrast tested?

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] Which Seattle Humane animal would you like to dedicate this PR to?
Lucy!
LUCY

### CODE REVIEW

File: src/services/AnimalService.ts
Comment (DillonOLeary) [inline src/services/AnimalService.ts:1]: If you remove this code, is the behavior preserved somewhere else in the code?

File: src/services/ClientService.ts
Comment (DillonOLeary) [inline src/services/ClientService.ts:1]: Same with this. Where was the behavior moved to?

**Review by DillonOLeary — COMMENTED** [review]
Thanks for cleaning stuff up. I pushed 2 slight changes but this looks good generally. I also had some questions as a part of my review

**Comment by DillonOLeary** [comment]
Also love the animal photo :)

**Review by DillonOLeary — APPROVED** [review]

File: src/services/AnimalService.ts
Comment (sharrief) [reply src/services/AnimalService.ts:1]: I know we discussed offline but just adding here for the history: I removed the code as it was basically placeholder code that we have now replaced with full implementations. We have Supabase generated types not that replaces the placeholders, and we have proper queries for tickets now as well.

**Comment by sharrief** [comment]
Thanks Dillon for the review!

**Pull request merged**